chalk/__init__.py

```diff
@@ -216,11 +217,12 @@ def beside(diagram1: Diagram, diagram2: Diagram) -> Diagram:
     Args:
         diagram1 (Diagram): Left diagram object.
         diagram2 (Diagram): Right diagram object.
+        direction (Optional[Vec2]): Placement direction.
```

Comment:
The documented type (`Optional[Vec2]`) doesn't agree with the given type (`Vec2`).